.speakeasy/overlay.mcp.yaml

```
@@ -13,6 +13,7 @@ actions:
       x-speakeasy-mcp:
         disabled: false
         name: list-retros-by-incident
+        scopes: "read", "write"
```

I think this is just a read operation

```suggestion
        scopes: "read"
```